src/data/gods/artemis.ts

```diff
@@ -1,56 +1,17 @@
-import { COMMON, RARE, EPIC, HEROIC, DUO } from "./rarities";
-import { abilityFormatter } from "./formatters";
-import { ATTACK, SPECIAL, CAST, DASH, OTHER } from "./abilityTypes";
 import { mapValues, toArray } from "lodash";
+import { CAST, DASH, OTHER } from "./abilityTypes";
+import { AIR, EARTH } from "./elements";
+import { abilityFormatter } from "./formatters";
 import { Boon, God } from "./god";
-import { EARTH, AIR, } from "./elements";
+import { COMMON, EPIC, RARE } from "./rarities";
 
 const info =
   "Artemis, Goddess of the Hunt. Her powers cause critical hits and create seeking projectiles.";
 
-const attack: Boon = {
-  name: "Unknown",
-  type: ATTACK,
-  info: (value) => `Unknown`,
-  values: {
-    [COMMON]: {
-      1: "20%",
-    },
-    [RARE]: {
-      1: `${20 * 1.3}-${20 * 1.5}`,
-    },
-    [EPIC]: {
-      1: `${20 * 1.8}-${20 * 2}`,
-    },
-    [HEROIC]: {
-      1: `${20 * 2.3}-${20 * 2.5}`,
-    },
-  },
-};
-
-const special: Boon = {
-  name: "Unknown",
-  type: SPECIAL,
-  info: (value) => `Unknown`,
-  values: {
-    [COMMON]: {
-      1: "40%",
-    },
-    [RARE]: {
-      1: `${40 * 1.3}-${40 * 1.5}`,
-    },
-    [EPIC]: {
-      1: `${40 * 1.8}-${40 * 2}`,
-    },
-    [HEROIC]: {
-      1: `${40 * 2.3}-${40 * 2.5}`,
-    },
-  },
-};
-
-const cast: Boon = {
+const easyShot: Boon = {
   name: "Easy Shot",
   type: CAST,
```

Comment:
Do any of Artemis' boons actually occupy the exclusive boon slots? Or does she work like Hermes now?

Reply:
They do not, she works like Hermes now